sdk/examples/how_tos/advanced_transactions/claim_transaction.rs

```diff
@@ -16,31 +16,32 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
-
-    // Get the account we generated with `create_account`
-    let account = wallet.get_account("Alice").await?;
+    // Get the wallet we generated with `create_wallet`.
```

Comment:
No create_wallet and get anymore :trollface:

Reply:
true that

Reply:
The function is not called like this